src/Products/GenericSetup/interfaces.py

```diff
@@ -512,6 +514,13 @@ def runImportStepFromProfile(profile_id, step_id,
         o If 'run_dependencies' is True, then run any out-of-date
           dependency steps first.
 
+        o 'path' is the path to a directory with files to import.
+          This avoids the need to register a full profile, when you only need
+          to import a few files once in an upgrade step.
+          It is best to use an absolute path, otherwise it would depend on
+          your working directory.
+          Note that metadata.xml is never read, so you cannot use dependencies.
```

Comment:
If you keep the *path* parameter, I would remove the dependency note at this place. At this place we speak about  steps and step dependencies; the note refers to profile dependencies.

Reply:
That is confusing here yes. Removed.